.gitignore

```diff
@@ -2,3 +2,4 @@ build.log
 build.log.1
 tmp.repo/
 *.p5m.final
+build/python36/local.mog
```

Comment:
Why not add this to a .gitignore file inside build/python36 rather than at the top level?